concent_api/core/models.py

```diff
@@ -684,3 +685,45 @@ class PendingEthereumTransaction(Model):
     s = DecimalField(max_digits=78, decimal_places=0)
 
     created_at = DateTimeField(auto_now_add=True)
+
+
+class DepositAccount(Model):
+    client = ForeignKey(Client)
+    ethereum_public_key = CharField(max_length = ETHEREUM_ADDRESS_LENGTH)
```

Comment:
There is no need now to add spaces around `=`.

Reply:
shouldn't `ethereum_public_key` have **exactly** the same length as `ETHEREUM_ADDRESS_LENGTH`, not max?

Reply:
Done.